lexers/circular/php.go

```diff
@@ -17,7 +21,21 @@ var PHP = internal.Register(MustNewLexer(
 		EnsureNL:        true,
 	},
 	phpCommonRules.Rename("php", "root"),
-))
+).SetAnalyser(func(text string) float32 {
+	matched, _ := shebang.MatchString(text, "php")
+	if matched {
+		return 1.0
+	}
+
+	var result float32 = 0
+
+	matched, _ = phpAnalyserRe.MatchString(text)
+	if matched {
+		result += 0.3
```

Comment:
actually we can return here directly. No need actually to have `result` variable.
```suggestion
		return 0.3
```